proxy-jsonrpc/proxy_test.go

```diff
@@ -0,0 +1,27 @@
+package proxy_test
+
+import (
+	"context"
+	"testing"
+
+	"github.com/stretchr/testify/require"
+
+	"github.com/rollkit/go-da/proxy-jsonrpc"
+	"github.com/rollkit/go-da/test"
+)
+
+func TestProxy(t *testing.T) {
+	dummy := test.NewDummyDA()
+	server := proxy.NewServer("localhost", "3450", dummy)
+	err := server.Start(context.TODO())
```

Comment:
Why are we using `context.TODO()` here and in line 19 vs a common context?

Reply:
Using `context.Background` instead.